job-service/src/main/java/com/hpe/caf/services/job/api/generated/model/Policy.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2016-2021 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hpe.caf.services.job.api.generated.model;
+//
+// Source code recreated from a .class file by IntelliJ IDEA
```

Comment:
Where did this file come from?

Reply:
I am not entirely sure, but from what I found, this could have been recreated if at a point I deleted it and it was still referenced elsewhere in the code. The IDE would recreate it from compile sources.
Maybe more likely, there was an issue with some cache being corrupt. I cleared those to make sure.